lib/src/over_react_test/wrapper_component.dart

```diff
@@ -14,30 +14,16 @@
 
 import 'package:over_react/over_react.dart';
 
-// ignore: uri_has_not_been_generated
 part 'wrapper_component.over_react.g.dart';
 
 /// A helper component for use in tests where a component needs to be
 /// rendered inside a wrapper, but a composite component must be used
 /// for compatability with `getByTestId()`.
-@Factory()
-// ignore: undefined_identifier
-UiFactory<UiProps> Wrapper =
-    // ignore: undefined_identifier
-    _$Wrapper;
+UiFactory<WrapperProps> Wrapper = _$Wrapper; // ignore: undefined_identifier
 
-@Props()
-class _$WrapperProps extends UiProps {}
+mixin WrapperProps on UiProps {}
```

Comment:
I updated `WrapperProps` without deprecating it because [nothing ever extends from it](https://sourcegraph.wk-dev.wdesk.org/search?q=lang:dart+extends%5Cs%2B%28WrapperProps%29+count:1000&patternType=regexp).